.nvmrc

```diff
@@ -0,0 +1 @@
+14.18.1
```

Comment:
Should the version be 18?

Reply:
I placed it based on the version in docker image. I think we need to have followup stories to update the node version.